post_processing/README.md

```diff
@@ -0,0 +1,61 @@
+# Post-processing / Regression tests
+
+This document outlines how the post-processing and regression
+test infrastructure works, including
+
+ - How regression tests work
+ - How to approve pull requests that fail regression tests
+ - Other failures
+
+## How the regression tests work
+
+Regression tests are performed at the end of `examples/hybrid/driver.jl`, after a simulation completes, and relies on a unique job id (`job_id`). Here is an outline of the regression test procedure:
+
+ 0) Run a simulation, with a particular `job_id`, to the final time.
+ 1) Load a dictionary, `all_best_mse`, of previous "best" mean-squared errors from `mse_tables.jl` and extract the mean squared errors for the given `job_id` (store in job-specific dictionary, `best_mse`).
+ 2) Export the solution (a `FieldVector`) at the final simulation time to an `NCDataset` file.
+ 3) Compute the errors between the exported solution (in step 2) and the exported solution from the reference `NCDataset` file (which is saved in a dedicated folder on the Caltech Central cluster) and save into a dictionary, called `computed_mse`.
+ 4) Export this dictionary (`computed_mse`) to the output folder
+ 5) Test that `computed_mse` is no worse than `best_mse` (determines if regression test passes or not).
+
+After these steps are performed at the end of the driver, additional jobs are run:
+
+ 1) Print `computed_mse` for all jobs to make updating `post_processing/mse_tables.jl` easy
+ 2) If we're on the staging branch (all tests have passed, and the PR is effectively merging), move the `NCDataset`s from the scratch directory onto the dedicated folder on the Caltech Central cluster.
+
+## How to approve pull requests (PR) that fail regression tests
```

Comment:
maybe "How to merge pull requests (PR) that get approved but fail regression tests" since a failure doesn't stop people from hitting the approve button